src/plugins/media-analysis/media-analysis.yml

```diff
@@ -18,5 +18,5 @@ mediaAnalysis:
     en: You can drop a CSV file in this box in order to import segments
     de: Sie können eine CSV-Datei in diese Box ziehen, um Segmente zu importieren
   startPlaybackFromHere:
-    en: Click here to play from this part
-    de: Klicke hier, um die Wiedergabe ab diesem Teil zu starten
+    en: Play from this part
+    de: Die Wiedergabe ab diesem Teil starten
```

Comment:
We should scratch the "Die" or - closer to the English text - have: "Ab diesem Teil abspielen"

Reply:
Changed to "Ab diesem Teil abspielen", thank you.